@xen-orchestra/web-core/lib/components/ui/button/UiButton.vue

```diff
@@ -0,0 +1,472 @@
+<!-- v4 -->
+<template>
+  <button :class="classNames" :disabled="busy || isDisabled" class="button" type="button">
+    <VtsIcon :busy :icon="leftIcon" accent="current" class="icon" fixed-width />
+    <slot />
+    <VtsIcon :icon="rightIcon" accent="current" class="icon" fixed-width />
+  </button>
+</template>
+
+<script lang="ts" setup>
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import { useContext } from '@core/composables/context.composable'
+import { DisabledContext } from '@core/context'
+import { toVariants } from '@core/utils/to-variants.util'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { computed } from 'vue'
+
+type ButtonVariant = 'primary' | 'secondary' | 'tertiary'
+type ButtonAccent = 'brand' | 'success' | 'warning' | 'danger'
```

Comment:
According to the design system, the variant `brand` should be replaced by `info`.
Remember to update the CSS class names accordingly, and also the component usages.
```suggestion
type ButtonAccent = 'info' | 'success' | 'warning' | 'danger'
```